### DISCUSSION TITLE
第3节课笔记(4班)

### DISCUSSION THREAD

**Discussion by FlamingoPg** [InternLM/Tutorial#201] [笔记]
请大家把笔记链接交到这里哦~并且写下群昵称方便我和班长打分

**Comment by zhjunqin** [comment]
zhjunqin: https://blog.csdn.net/zhjunqin/article/details/135491992?spm=1001.2014.3001.5502

**Comment by onlyzer0** [comment]
https://blog.csdn.net/qq_42907812/article/details/135512101?spm=1001.2014.3001.5502
onlyzer0

**Comment by phbst** [comment]
https://thankful-archeology-738.notion.site/dea91be950004ddb8a2db9b3d10e5344?pvs=4
